src/Microsoft.DocAsCode.Build.Engine/PostbuildPhaseHandlerWithIncremental.cs

```diff
@@ -88,6 +92,28 @@ private void ReloadModelsPerChanges(IEnumerable<HostService> hostServices)
             }
         }
 
+        private void RegisterUnloadedXRefSpec()
+        {
+            var lastXrefMap = LastBuildVersionInfo?.XRefSpecMap;
+            foreach (var m in _unloadedFiles)
+            {
+                if (lastXrefMap == null)
+                {
+                    throw new BuildCacheException($"Full build hasn't loaded XRefMap.");
+                }
+                IEnumerable<XRefSpec> specs;
+                if (!lastXrefMap.TryGetValue(m, out specs))
```

Comment:
if an overwrite file not get loaded,  i'll add an empty record in buildversioninfo.XrefMap

---
In reply to: [90374917](https://github.com/dotnet/docfx/pull/936#discussion_r90374917) [](ancestors = 90374917,90229696)